chainer/functions/evaluation/classification_summary.py

```diff
@@ -75,12 +75,13 @@ def classification_summary(y, t, label_num=None, beta=1.0, ignore_label=-1):
     This function calculates the following quantities for each class.
 
     - Precision: :math:`\\frac{\\mathrm{tp}}{\\mathrm{tp} + \\mathrm{fp}}`
-    - Recall: :math:`\\frac{\\mathrm{tp}}{\\mathrm{tp} + \\mathrm{tn}}`
+    - Recall: :math:`\\frac{\\mathrm{tp}}{\\mathrm{tp} + \\mathrm{fn}}`
     - F beta Score: The weighted harmonic average of Precision and Recall.
     - Support: The number of instances of each ground truth label.
 
-    Here, ``tp``, ``fp``, and ``tn`` stand for the number of true positives,
-    false positives, and true negative, respectively.
+    Here, ``tp``, ``fp``, ``tn``, and ``fn`` stand for the number of true
```

Comment:
Would you like to mention the number of true negatives (`tn`), while it doesn't appear in the formulas?

Reply:
Good point. I think it's possible to remove mention of `tn`. However, one could always keep it for the sake of completeness or in case other equations for other metrics are added later.

I don't have strong feelings about it either way.

The more essential part of this PR is to fix the typo in the equation above.